packages/js/experimental-products-app/package.json

```
@@ -35,6 +35,7 @@
 		"access": "public"
 	},
 	"dependencies": {
+		"@base-ui/react": "1.4.1",
```

Biome (2.4.15)

[error] The key `@base-ui/react` was already declared.

(lint/suspicious/noDuplicateObjectKeys)